jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy

```diff
@@ -52,6 +52,8 @@ pipeline {
 
         string(name: 'CORTX_SCRIPTS_REPO', defaultValue: 'https://github.com/Seagate/cortx-k8s', description: 'Repository for cortx-k8s scripts (Services Team)', trim: true)
         string(name: 'CORTX_SCRIPTS_BRANCH', defaultValue: 'cortx-test', description: 'cortx-k8s scripts (Provisioner Team)', trim: true)
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'https://github.com/aayushisharma88/cortx-re', description: 'Repository for cortx-re scripts (Services Team)', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'UPGRADE_SANITY', description: 'Repository for cortx-re scripts (Services Team)', trim: true)
```

Comment:
please change branch to `main`

Reply:
@gauravchaudhari02 , @aayushisharma88  - Can you please resolve this thread